cli/internal/util/parse_concurrency_test.go

```diff
@@ -53,17 +54,17 @@ func TestParseConcurrency(t *testing.T) {
 	}
 
 	t.Run("throw on invalid string input", func(t *testing.T) {
-		_, err := ParseConcurrency("asdf")
+		_, err := parseConcurrency("asdf")
```

Comment:
Can you do me a favor and add a few test cases I know will trigger edge cases?

- `"infinity%"`
- `"-infinity%"`
- `"nan%"`
- `0b01`
- `0o644`
- `0xFF`
- `0644`

Yes, clearly this is a "don't do this," but I want to make sure we don't panic.

https://cs.opensource.google/go/go/+/refs/tags/go1.18.3:src/strconv/atof.go;l=42-72;drc=55590f3a2b89f001bcadf0df6eb2dde62618302b

Reply:
https://go.dev/play/p/5I-zD8uGVVK

Reply:
Done. We allow `0644`, but parse it as decimal. I don't think that's worth fixing currently, since `strconv.ParseFlot` doesn't handle it for us.